mkdocs.yml

```diff
@@ -44,7 +44,7 @@ nav:
     - AEA and web frameworks: 'aea-vs-mvc.md'
     - Build your first skill - search & discovery: 'skill-guide.md'
     - Core components - Part 2: 'core-components-2.md'
-    - How AEAs talk to each other - interaction protocols: 'interaction-protocol.md'
```

Comment:
Please revert

Reply:
I am not really a fan of long sentences in titles, but reverted.